src/AzureStack/NetworkAdmin/Network.Admin/Microsoft.AzureStack.Management.Network.Admin.csproj

```diff
@@ -0,0 +1,15 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@deathly809 missing comment around the includes.
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L2
Also missing the include that is required in the footer of the file, model your csproj using this
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L22

Reply:
Fixed.